### ISSUE TITLE
run-script with user input

### ISSUE THREAD

**Issue by higoka** [composer/composer#8301]
Is it possible to run-script a php file that requires user input?

For example:  **test.php**
```php
<?php

echo 'type something: ';
echo 'you typed: ' . fgets(STDIN);
```
In **composer.json**:
```json
"scripts": {
    "test": "php -f test.php"
}
```
When i run: `$ composer run-script test`
```
> php -f test.php
type something: you typed: 
```
I cant input anything.

Using: `$ php -f test.php`
```
type something: foo
you typed: foo
```

**Comment by higoka** [comment]
any updates?

**Comment by Mario-F** [comment]
any updates?

**Comment by tlamy** [comment]
Ping? 
Same for me, `$answer = trim(fgets(STDIN));` always returns false when called within a `composer create-project` script

**Comment by Seldaek** [comment]
This should be fixed on linux and in composer 2.0, you can try using snapshots (`composer self-update --snapshot`)
